pkg/s3utils/s3utils.go

```diff
@@ -79,7 +79,22 @@ func GetAWSDetailsFromEnv() (id string, secret string, endpoint string,
 	expect(err).NotTo(haveOccurred(),
 		fmt.Sprintf("S3_DISABLE_SSL=%s is not a valid boolean value", disableSSL))
 
-	return id, secret, endpoint, s3Region, disableSSLBool
+	sseType = os.Getenv("S3_SSE_TYPE")
```

Comment:
Convert this to a custom type and have constant strings defined for `S3_SSE_TYPE` and `S3_ENCRYPTION_POLICY`

Reply:
done